jenkins/pipelines/cd/tiup/tiup-check-online-version.groovy

```diff
@@ -5,18 +5,33 @@ tiup mirror reset
 tiup install tikv:${params.VERSION} tidb:${params.VERSION} pd:${params.VERSION} tiflash:${params.VERSION} cdc:${params.VERSION}
 """
 
+final  tiupYaml='''
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: tiup
+    image: hub.pingcap.net/jenkins/tiup
+    args: ["sleep", "infinity"]
+'''
```

Comment:
Prefer to use yaml file to manage them.

And we need set tty enabled, no need args.

Reply:
it will fail
and tty will use resource on host